apps/full-site-editing/full-site-editing-plugin/premium-content/blocks/container/index.js

```diff
@@ -94,6 +105,48 @@ const settings = {
 		// Removes support for an HTML mode.
 		html: false,
 	},
+	transforms: {
+		from: [
+			{
+				type: 'block',
+				isMultiBlock: true,
+				blocks: [ '*' ],
+				__experimentalConvert( blocks ) {
+					// Avoid transforming any premium-content blocks
+					if ( blocks.some( blockContainsPremiumBlock ) ) {
+						return;
+					}
+
+					// Clone the Blocks
+					// Failing to create new block references causes the original blocks
+					// to be replaced in the switchToBlockType call thereby meaning they
+					// are removed both from their original location and within the
+					// new premium content block.
+					const innerBlocksSubscribe = blocks.map( ( block ) => {
+						return createBlock( block.name, block.attributes, block.innerBlocks );
+					} );
+
+					const innerBlocksLoggedOut = [
+						createBlock( 'core/heading', {
+							content: __( 'Subscribe to get access', 'premium-content' ),
+							level: 3,
+						} ),
+						createBlock( 'core/paragraph', {
+							content: __(
+								'Read more of this content when you subscribe today.',
+								'premium-content'
+							),
+						} ),
+					];
```

Comment:
I wonder if there is any way we can fall back to the [default template](https://github.com/Automattic/wp-calypso/blob/292c8c3bbef807b589be3e10a16cb5d78e6a3ea0/apps/full-site-editing/full-site-editing-plugin/premium-content/blocks/logged-out-view/edit.js#L62-L76) defined on the block.

Maybe calling `createBlock` without any inner block?

Reply:
> Maybe calling `createBlock` without any inner block?

Yup, that worked: 0c1cef5.